sparse/_compressed/compressed.py

```diff
@@ -810,6 +810,42 @@ def _prune(self):
         else:
             self.indices = self.indices[mask]
 
+    def pad(self, pad_width, mode="constant", **kwargs):
+        """
+        Performs the equivalent of :obj:`numpy.pad` for :obj:`GCXS`. Note that
+        this function returns a new array instead of a view.
+
+        Parameters
+        ----------
+        pad_width : {sequence, array_like, int}
+            Number of values padded to the edges of each axis. ((before_1, after_1), … (before_N, after_N)) unique pad widths for each axis. ((before, after),) yields same before and after pad for each axis. (pad,) or int is a shortcut for before = after = pad width for all axes.
+
+        mode : str
+            Pads to a constant value which is fill value. Currently only constant mode is implemented
+
+        constant_values : int
+            The values to set the padded values for each axis. Default is 0. This must be same as fill value.
+
+        Returns
+        -------
+        GCXS
+            The padded sparse array.
+
+        Raises
+        ------
+        NotImplementedError
+            If mode != 'constant' or there are unknown arguments.
+
+        ValueError
+            If constant_values != self.fill_value
+
+        See Also
+        --------
+        :obj:`numpy.pad` : NumPy equivalent function
+
+        """
+        return pad(self, pad_width, mode=mode, **kwargs)
```

Comment:
No need for this function at all. `np.pad` calls `__array_function__` which works just fine with `sparse.pad`